asciidoc-confluence-publisher-docker/Dockerfile

```diff
@@ -29,4 +30,5 @@ ENTRYPOINT ["sh", "-c", "java -jar /opt/asciidoc-confluence-publisher-docker.jar
     \"pageTitleSuffix=$PAGE_TITLE_SUFFIX\" \
     \"publishingStrategy=$PUBLISHING_STRATEGY\" \
     \"versionMessage=$VERSION_MESSAGE\" \
+    \"attributes=$ATTRIBUTES\" \
```

Comment:
We miss a test that is actually passing user attributes via env var to the Docker-based publishing. As we don't really have a proper integration testing environment specifically for such tests, the only quick way I see is a "smoke test" based on `publishWithDockerImage.sh` (i.e. passing some - actually unused - attributes). That would at least protect against issues with double-quotes escaping.

Reply:
That is actually related to the comment https://github.com/confluence-publisher/confluence-publisher/pull/139/files#r267508581 - we do not have a single integration (or at least "smoke") test that ensures that when using the Docker-based publishing, the user attributes are properly passed and replaced.

As I said, we currently do not have a proper integration testing harness that allows us to publish via Docker and then verify certain aspects of the published documentation. One approach might be to trigger the publishing from a Java integration test e.g. using testcontainers. Anyway, that's out of scope of this pull request. 

So for now, let's keep it as you have it, we are not making things worse. I will take note of the need for enhancing the Docker-based publishing integration tests.